draft-ietf-taps-impl.md

```diff
@@ -968,7 +968,7 @@ Initiate:
 : Calling `Initiate` on a TLS Connection causes it to first initiate a TCP connection. Once the TCP protocol is Ready, the TLS handshake will be performed as a client (starting by sending a `client_hello`, and so on).
 
 InitiateWithSend:
-: Early idempotent data is supported by TLS 1.3, and sends encrypted application data in the first TLS message when performing session resumption. For older versions of TLS, or if a session is not being resumed, the initial data will be delayed until the TLS handshake is complete. TCP Fast Option can also be enabled automatically.
+: Early safely replayable data is supported by TLS 1.3, and sends encrypted application data in the first TLS message when performing session resumption. For older versions of TLS, or if a session is not being resumed, the initial data will be delayed until the TLS handshake is complete. TCP Fast Option can also be enabled automatically.
 
```

Comment:
TCP Fast Option should be TCP Fast Open I think.

Reply:
Fixed; I almost hesitated to fix it, we could have left it in there as an Easter Egg of some kind. An April 1 (2021) RFC could then have followed up and fully spec'ed the TCP Fast Option...